website/vercel.json

```diff
@@ -2,6 +2,11 @@
   "cleanUrls": true,
   "trailingSlash": false,
   "redirects": [
+    {
```

Comment:
@nghi-ly also added this redirect as the faqs are also probably linked on other sites (e.g. here: https://arc.net/l/quote/mdupwkgf)